src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,56 @@
+import { v4 } from 'uuid';
+
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+const books = [
+  {
+    author: 'Author 1',
+    title: 'Book 1',
+    category: 'Category A',
+    id: v4(),
+  },
+  {
+    author: 'Author 2',
+    title: 'Book 2',
+    category: 'Category B',
+    id: v4(),
+  },
+  {
+    author: 'Author 3',
+    title: 'Book 3',
+    category: 'Category C',
+    id: v4(),
+  },
+];
+
+const bookReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        action.content,
+      ];
+    case REMOVE_BOOK:
+      return state.filter((book) => (book.id !== action.id));
+    default: return books || state;
+  }
+};
+
+export const addnewBook = (id, title, author) => ({
+  type: ADD_BOOK,
+  content: {
+    id,
+    title,
+    author,
+  },
+});
+
+export const removeBook = (id) => ({
```

Comment:
- [ ] Please as stated in the requirement, kindly use Redux Toolkit to create a slice for managing a list of books in your Redux store. This [tutorial might be of help](https://redux-toolkit.js.org/tutorials/quick-start#create-a-redux-state-slice). 

<img width="651" alt="Untitled" src="https://github.com/Kwamay/React-Books/assets/103386183/a41d0429-1b6b-46eb-aee1-23d23eccfd1c">